frontend/src/components/LandingPage/Features/Features.js

```diff
@@ -0,0 +1,88 @@
+/* 
+    This component will have a title and a description of the features 
+    Features include: 
+        - snap a picture or manually enter ingredients
+        - getting recipes with ingredients you have
+        - seeing nutritional information
+        - and getting videos to go along with your recipe
+*/
+
+import React from 'react';
+import { Card, CardImg, CardTitle, CardText, CardDeck, CardSubtitle, CardBody, Jumbotron } from 'reactstrap';
+import './Features.css'
+import camera from './assets/coolcamera.jpeg';
+import nutritional from './assets/nutritional.jpg';
+import recipes from './assets/goodrecipe.jpeg';
+import videos from './assets/videos.jpg';
```

Comment:
Passing on some feedback I got from Miguel earlier on my tools component images!

Images could be moved to the public folder and then you can just have the filepath like `src='/coolcamera.jpeg'` or maybe you could have something like `const camera = '/coolcamera.jpeg'` if you wanted to keep the `img = {camera}` down below as is.

Reply:
That sounds great! I'll implement those changes

Reply:
Awesome! Just approved your changes!